Makefile

```diff
@@ -8,7 +8,7 @@ MCUS			= H L
 LAYOUTS_X		= A B C
 MCUS_X		= X
 DEADTIMES		= 0 5 10 15 20 25 30 40 50 70 90 120
-PWM_FREQS		= 24 48 96
+PWM_FREQS		= 24 48 96 DYNAMIC
```

Comment:
See comment in the build file, this should no longer be needed, since we have unified the builds and they are agnostic of PWM frequency anyway.

Reply:
Do you want to leave only 24 khz option?

Reply:
I am not sure if we need to leave an option at all. We build it by default with 24kHz and people can then set it up in the configurator as they see fit. So the hex files simply no longer will have the PWM distinction in their name.

Reply:
done